controllers/argocd/argocd_controller.go

```diff
@@ -211,6 +211,6 @@ func (r *ReconcileArgoCD) Reconcile(ctx context.Context, request ctrl.Request) (
 // SetupWithManager sets up the controller with the Manager.
 func (r *ReconcileArgoCD) SetupWithManager(mgr ctrl.Manager) error {
 	bldr := ctrl.NewControllerManagedBy(mgr)
-	r.setResourceWatches(bldr, r.clusterResourceMapper, r.tlsSecretMapper, r.namespaceResourceMapper, r.clusterSecretResourceMapper, r.applicationSetSCMTLSConfigMapMapper)
+	r.setResourceWatches(bldr, r.clusterResourceMapper, r.tlsSecretMapper, r.namespaceResourceMapper, r.sourceNamespacemapper, r.clusterSecretResourceMapper, r.applicationSetSCMTLSConfigMapMapper)
```

Comment:
Follow the variable naming convention followed in this file. It should be `sourceNamespaceMapper` instead of `sourceNamespacemapper`